scripts/patchup-example-names.mjs

```diff
@@ -1,17 +1,28 @@
 import fs from "fs/promises";
 import path from "path";
 
-let dirPath = path.join(process.cwd(), "examples");
-let dir = await fs.readdir(dirPath);
+let examplePath = path.join(process.cwd(), "examples");
+let templatePath = path.join(process.cwd(), "templates");
+let examples = await fs.readdir(examplePath);
+let templates = await fs.readdir(templatePath);
 
-dir.forEach(async (name) => {
-  let fullPath = path.join(dirPath, name);
+async function update(dir, name) {
+  let base = path.basename(dir).slice(0, -1);
+  let fullPath = path.join(dir, name);
   let stat = await fs.stat(fullPath);
   if (stat.isDirectory()) {
     let pkgPath = path.join(fullPath, "package.json");
     let pkg = await fs.readFile(pkgPath);
     let obj = JSON.parse(pkg);
-    obj.name = `remix-example-${name}`;
+    obj.name = `remix-${base}-${name}`;
     await fs.writeFile(pkgPath, JSON.stringify(obj, null, 2) + "\n");
```

Comment:
We may as well use `sortPackageJson(obj)` here instead of `JSON.stringify`.

See https://www.npmjs.com/package/sort-package-json